api/v1beta1/kafkacluster_types.go

```diff
@@ -149,6 +154,10 @@ type EnvoyConfig struct {
 	Tolerations              []corev1.Toleration           `json:"tolerations,omitempty"`
 	Annotations              map[string]string             `json:"annotations,omitempty"`
 	LoadBalancerSourceRanges []string                      `json:"loadBalancerSourceRanges,omitempty"`
+	BringYourOwnLB           bool                          `json:"bringYourOwnLoadBalancer,omitempty"`
```

Comment:
i would rename this to `CreateLoadBalancer`

Reply:
I agree that it would be a better naming, but the default value (if property is missing is `false`), and in order to make this change backwards compatible we should explicitly ask **not** to create the LB. Hence, the  `BringYourOwnLB`

Reply:
i think we can create default value as true right?

Reply:
It might be a bit tricky. I will take a look and see if we can do it.